client/src/templates/Challenges/redux/completion-epic.js

```diff
@@ -183,30 +174,8 @@ export default function completionEpic(action$, state$) {
   );
 }
 
-async function findPathToNavigateTo(
-  certification,
-  nextChallengePath,
-  superBlock,
-  state,
-  challengeType
-) {
+async function findPathToNavigateTo(nextChallengePath, superBlock) {
   let canClaimCert = false;
-  const isProjectSubmission = [
-    challengeTypes.frontEndProject,
-    challengeTypes.backEndProject,
-    challengeTypes.pythonProject
-  ].includes(challengeType);
-  if (isProjectSubmission) {
-    const username = usernameSelector(state);
-    try {
-      const response = await getVerifyCanClaimCert(username, certification);
-      if (response.status === 200) {
-        canClaimCert = response.data?.response?.message === 'can-claim-cert';
-      }
-    } catch (err) {
-      console.error('failed to verify if user can claim certificate', err);
-    }
-  }
   let pathToNavigateTo;
 
   if (nextChallengePath.includes(superBlock) && !canClaimCert) {
```

Comment:
```suggestion
  if (nextChallengePath.includes(superBlock)) {
```
Presumably.

Also, the `else if (canClaimCert) {` branch is always false now.